assets/javascripts/initializers/discourse-reactions.js.es6

```diff
@@ -24,6 +24,33 @@ function initializeDiscourseReactions(api) {
 
   api.replaceIcon("notification.reaction", "discourse-emojis");
 
+  api.modifyClass("component:scrolling-post-stream", {
+    didInsertElement() {
+      this._super(...arguments);
+
+      const topicId = this?.posts?.firstObject?.topic_id;
+      if (topicId) {
+        this.messageBus.subscribe(`/topic/${topicId}/reactions`, (data) => {
+          this.dirtyKeys.keyDirty(
+            `discourse-reactions-counter-${data.post_id}`,
+            {
+              onRefresh: "reactionsChanged",
+              refreshArg: data,
+            }
+          );
+          this._refresh({ id: data.post_id });
+        });
+      }
+    },
+
+    unsusbcribe() {
```

Comment:
This function doesn't seem to be called at all since core does not have an `unsubscribe` function defined.

Reply:
yes indeed, it used to be in topic controller but we changed it to be able to use dirtyKeys.

Will see what I can do to make it called.